base/collect_database.go

```diff
@@ -215,28 +219,6 @@ func (t *collectDatabaseTask) run(exp *Experiment) error {
 	// initialize defaults
 	t.initializeDefaults()
 
-	// get current directory path
-	path, err := os.Getwd()
-	if err != nil {
-		return err
-	}
-
-	// collect all files paths in current directory that ends with metrics.yaml
-	metricFilePaths := []string{}
-	err = filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
-		if err != nil {
-			return err
-		}
-
-		if !info.IsDir() && strings.HasSuffix(path, "metrics.yaml") {
```

Comment:
The old method of identifying metrics files is to iterate through all file paths and look for paths that end with `metrics.yaml`.